src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskStateSerializer.scala

```diff
@@ -0,0 +1,118 @@
+package mesosphere.marathon.core.task.tracker.impl
+
+import mesosphere.marathon.Protos.MarathonTask
+import mesosphere.marathon.core.task.tracker.TaskState
+import mesosphere.marathon.state.Timestamp
+import org.apache.mesos.{ Protos => MesosProtos }
+
+/**
+  * Converts between [[TaskState]] objects
+  * and their serialized representation: MarathonTask.
+  */
+object TaskStateSerializer {
```

Comment:
Would be nice to have a type class approach like Play Json Reader/Writer for all our proto stuff.